web/src/app/modules/views/side/modules/selected-element/services/selected-element.service.ts

```diff
@@ -20,36 +22,89 @@ export class SelectedElementService {
         });
     }
 
-    public get selectionChanged(): EventEmitter<IContainer> {
+    public get selectionChanged(): EventEmitter<IContainer[]> {
         if (!this._selectionChanged) {
-            this._selectionChanged = new EventEmitter<IContainer>();
+            this._selectionChanged = new EventEmitter<IContainer[]>();
         }
         return this._selectionChanged;
     }
 
     public get hasSelection(): boolean {
-        return this.selectedElement !== undefined;
+        return this._selectedElements.length > 0;

```

Comment:
can _selectedelements still be undefined?

Reply:
No it is initialized as an empty list and the setter doesn't allow it to be set to undefined.